plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -392,20 +358,27 @@ class NewCreateServiceModalForm extends Component {
   }
 
   getSectionContent(data, errorMap) {
-    const errorsAlertComponent = this.getErrorsAlertComponent();
+    const {showAllErrors} = this.props;
+    const errors = this.getErrors();
 
     if (this.state.isPod) {
       return [
         <TabView id="networking" key="multinetworking">
-          {errorsAlertComponent}
+          <ErrorsAlert
```

Comment:
I wonder if this should be added to our `<TabView/>` component and we pass the props to it? Could you ever see a case where we might want to configure where the `<ErrorsAlert/>` component is placed?